### PR TITLE
Update plugin documentation to be more concise.

### PR DESCRIPTION
**Pull request by Ziinc**
## Changes/Tasks

- [ ] Changed code

## Types of changes

- [ ] Refactoring/add tests (refactoring or adding test which isn't a fix or add a feature)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly.
- [ ] My changes have tests around them

### CODE REVIEW

File: docs/plugins/README.md [hunk 1]
```diff
@@ -77,24 +73,31 @@ export default {
 ```
 
 ## Plugin API
-
-Plugins at the end of the day, are just files (or a `directory` that contains the files):
-
+All plugins contains two at least files:
```
Comment (tannerlinsley) [inline docs/plugins/README.md]: All plugins contain at least one of these files:

File: docs/plugins/README.md [hunk 2]
```diff
@@ -77,24 +73,31 @@ export default {
...
+## How to write the `node.api.js` and `browser.api.js` files when creating plugins?
+The file (for either environment) must:
 
 - Provide a `function` as the `default export`
-- That function receives **plugin options from the user (optional)**
+- That function receives an optional **user plugin options** argument
 - **Return an `object`** providing any **API methods** to implement
 
-Here is a **pseudo** example of what a plugin typically looks like:
+E.g. of basic plugin:
```
Comment (tannerlinsley) [inline docs/plugins/README.md]: Basic plugin example:

File: docs/plugins/README.md [hunk 3]
```diff
@@ -77,24 +73,31 @@ export default {
...
+```javascript
+export default pluginOptions => ({
+  myMethod: options => {
+    console.log('hello world');
+  }
+})
 
+```
+E.g. of more complex plugin
```
Comment (tannerlinsley) [inline docs/plugins/README.md]: Do we need to examples?

File: docs/plugins/README.md [hunk 4]
```diff
@@ -110,6 +113,17 @@ export default pluginOptions => ({
...
 ```
 
+## What can plugins do?
+- [Modify your `static.config.js`](/docs/plugins/node-api.md#config-function)
+- [Transform your webpack config](/docs/plugins/node-api.md#webpack-functionfunction)
+- [Append JSX to the Head of the app](/docs/plugins/node-api.md#head-componentfunction)
+- [Customize your App's router](/docs/plugins/browser-api.md#router)
+- and more! 
+
+View the [browser api docs](/docs/plugins/browser-api.md) and the [node api docs](/docs/plugins/node-api.md) for full list of features.
```
Comment (tannerlinsley) [inline docs/plugins/README.md]: Capitalize API

File: docs/plugins/README.md [hunk 5]
```diff
@@ -110,6 +113,17 @@ export default pluginOptions => ({
...
+
 #### Plugins must be compiled if installed via node_modules
+Only the `plugins` directory will be transformed by react-static's babel runtime.
 
-If a plugin is installed via any method other than the `plugins` directory, it will not be transformed by react-static's `.babelrc` runtime, so you **must compile your plugin to be ES5 compatible if you distribute it**. This can be done via `@babel/core` and the babel-cli. The [react-static-plugin-styled-components](https://github.com/nozzle/react-static-plugin-styled-components) plugin does this and is a prime example to follow.
+Hence, when distributing your plugin, your plugin **must be ES5 compatible**.
+- E.g. of a plugin compiled before distribution is [react-static-plugin-styled-components](https://github.com/nozzle/react-static-plugin-styled-components).
```
Comment (tannerlinsley) [inline docs/plugins/README.md]: An example of a plugin compiled

File: docs/plugins/README.md [hunk 6]
```diff
@@ -1,13 +1,10 @@
 # Plugins
 
-React Static ships with a simple plugin API that allows both plugin creators and site developers to extend React-Static's core functionality.
+React Static ships with a plugin API to extend React-Static's functionality.
```
Comment (tannerlinsley) [inline docs/plugins/README.md]: No dash in React Static

File: docs/plugins/README.md [hunk 7]
```diff
@@ -32,19 +28,19 @@ If you simply need direct access to the the plugin API for a project, you can cr
...
   plugins: ['react-static-plugin-emotion', 'my-custom-plugin'],
 }
 ```
 
-#### Plugin Execution and Order
-
-**ORDER IS IMPORTANT!!!** Plugins are executed in the order that they are defined in the `plugins: []` array in the `static.config.js`. Also, any `node.api.js` and `browser.api.js` files found in the project root will be performed last.
+#### Plugin Execution and Order (IMPORTANT)
+Order of execution:
+1. Plugins in `plugins: []`, starting from first element of array.
```
Comment (tannerlinsley) [inline docs/plugins/README.md]: from the first

File: docs/plugins/README.md [hunk 8]
```diff
@@ -32,19 +28,19 @@ If you simply need direct access to the the plugin API for a project, you can cr
...
+2. Any `node.api.js` and `browser.api.js` files at project root.
```
Comment (tannerlinsley) [inline docs/plugins/README.md]: at the project

File: docs/plugins/README.md (hunk 3 above)
Comment (Ziinc) [reply docs/plugins/README.md]: guess not. seems redundant. I've consolidated both

Comment (tannerlinsley) [reply docs/plugins/README.md]: 👍

**Comment by tannerlinsley** [comment]
Awesome changes. Much better now. Thank you!

**Pull request merged**